.github/workflows/ci.yml

```diff
@@ -41,6 +41,9 @@ jobs:
       - name: lint
         run: npm run lint
 
+      - name: mypy
+        run: mypy --ignore-missing-imports --exclude py2 --exclude cs-env/ --exclude appengine_config.py .
+
```

Comment:
I set this to exclude directories that might not conform to standards by design or for reasons outside of our control.

Reply:
Also, can you define a script entry in package.json to run this?  I'd like to be able to run it locally easily.

Reply:
> I think our py2 directory was deleted. Is it still showing up in CI somehow?

Maybe I just have it on my machine locally and it's not actually available. I'll check!

> Also, can you define a script entry in package.json to run this? I'd like to be able to run it locally easily.

Definitely!